packages/amplify-cli/src/__tests__/plugin-helpers/verify-plugin.test.ts

```diff
@@ -0,0 +1,342 @@
+import { verifyPlugin, validPluginName } from '../../plugin-helpers/verify-plugin';
+import { JSONUtilities } from 'amplify-cli-core';
+import * as fs from 'fs-extra';
+import * as path from 'path';
+import { PluginVerificationError, PluginVerificationResult } from '../../domain/plugin-verification-result';
+import { PluginManifest } from '../../domain/plugin-manifest';
+import { AmplifyEvent } from '../../domain/amplify-event';
+
+const corePluginJson = {
+  name: 'core',
+  type: 'core',
+  commands: [
+    'categories',
+    'configure',
+    'console',
+    'delete',
+    'env',
+    'help',
+    'init',
+    'logout',
+    'migrate',
+    'plugin',
+    'publish',
+    'push',
+    'pull',
+    'run',
+    'status',
+    'uninstall',
+    'upgrade',
+    'version',
+  ],
+  commandAliases: {
+    h: 'help',
+    serve: 'run',
+    ls: 'status',
+  },
+};
+
+jest.mock('amplify-cli-core', () => ({
+  JSONUtilities: {
+    readJson: jest.fn(),
+    writeJson: jest.fn(),
+  },
+}));
+
+const fsMock = fs as jest.Mocked<typeof fs>;
+
+describe('verify-plugin', () => {
+  describe('verifyPlugin', () => {
+    beforeEach(() => {
+      const readJsonMock = JSONUtilities.readJson as jest.MockedFunction<typeof JSONUtilities.readJson>;
+      readJsonMock.mockClear();
+    });
+
+    it('returns PluginDirPathNotExist error when specify not exist path', async () => {
+      fsMock.pathExists.mockImplementation(() => Promise.resolve(false));
+      const result = await verifyPlugin('path/to/plugin');
```

Comment:
Can you make this change throughout the test file.

```suggestion
      const result = await verifyPlugin('path', 'to', 'plugin');
```

Reply:
Thank you for your suggestion. I will change it to use path.join function as below. 
```ts
const result = await verifyPlugin(path.join('path', 'to', 'plugin'));
```